protocol/x/clob/keeper/msg_server_cancel_orders.go

```diff
@@ -24,6 +24,16 @@ func (k msgServer) CancelOrder(
 ) (resp *types.MsgCancelOrderResponse, err error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
+	// Attach various logging tags relative to this request. These should be static with no changes.
+	ctx = log.AddPersistentTagsToLogger(ctx,
+		log.Module, log.Clob,
+		log.ProposerConsAddress, sdk.ConsAddress(ctx.BlockHeader().ProposerAddress),
+		log.Callback, lib.TxMode(ctx),
+		log.BlockHeight, ctx.BlockHeight(),
+		log.Handler, log.CancelOrder,
+		log.Msg, msg,
```

Comment:
is `msg` just a pointer in the resulting log? or do the fields get logged properly

Reply:
Looks fine to me
<img width="484" alt="Screenshot 2024-01-04 at 12 35 44 AM" src="https://github.com/dydxprotocol/v4-chain/assets/121899091/249ca1bf-d085-4f0f-87d2-ced6e9bf3195">